src/lib/Scenes/Home/Components/Sales/Components/SaleItem.tsx

```diff
@@ -2,17 +2,23 @@ import moment from "moment"
 import * as React from "react"
 import styled from "styled-components/native"
 
-import { Text, View } from "react-native"
+import { Dimensions, Text, View } from "react-native"
 import { createFragmentContainer, graphql } from "react-relay"
 
 import fonts from "../../../../../../data/fonts"
 import OpaqueImageView from "../../../../../Components/OpaqueImageView"
 import Serif from "../../../../../Components/Text/Serif"
 import { liveDate, timedDate } from "../formatDate"
 
+const screenSize = Dimensions.get("window")
+const isIPad = screenSize.width > 700
+const numColumns = isIPad ? 4 : 2
+const gutterSize = isIPad ? 100 : 60
+const containerWidth = (screenSize.width - gutterSize) / numColumns
```

Comment:
these can change at runtime (rotation, iOS window resizing), so I'd recommend putting them in render

Reply:
Good point